CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [v3.1.1] - 2021-06-28
+
+### Upgraded
+
+- build(deps): Bump go-storage to 4.2.0 (#48)
```

Comment:
ditto like https://github.com/beyondstorage/go-service-s3/pull/128#discussion_r660214115

And maybe we need to bump to `3.2.0` instead?